airflow/www/security.py

```diff
@@ -377,13 +406,13 @@ def has_all_dags_access(self):
         """
         Has all the dag access in any of the 3 cases:
         1. Role needs to be in (Admin, Viewer, User, Op).
-        2. Has can_dag_read permission on all_dags view.
-        3. Has can_dag_edit permission on all_dags view.
+        2. Has can_read permission on dags view.
+        3. Has can_edit permission on dags view.
```

Comment:
```suggestion
        2. Has can_read permission on AllDags view.
        3. Has can_edit permission on AllDags view.
```

(Not as part of this PR, but why on earth do we check the role membership at all? We should just use the permissions system!)

Reply:
Yeah permissions should be refactored/simplified in general, but I've been avoiding that as much as possible.